data/core/units/monsters/Cat_Tritail.cfg

```diff
@@ -0,0 +1,85 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Tritail Cat
+    name= _ "Tritail Cat"
+    race=cats
+    generate_name=no
+    image="units/monsters/cat/tritail.png"
+    profile="portraits/monsters/tritail-cat.webp"
+    hitpoints=42
+    movement=9
+    # similar to elves, but not as bad in caves, and not magical
+    movement_type=woodland
+    [defense]
+        cave=60
+        frozen=60
+    [/defense]
+    [movement_costs]
+        cave=2
+        mountains=2
+    [/movement_costs]
+    [resistance]
+        arcane=80
+    [/resistance]
+    experience=100
+    {AMLA_DEFAULT}
+    level=2
+    alignment=neutral
+    advances_to=null
+    cost=21
+    usage=scout
+    description=_ "Tritail are light-brown cats, roughly the same size as a wolf, that hunt the woods and hills at dusk and dawn.  Their most distinctive feature is their odd, triply branched tail, which they seem to use for camouflage and disruptive patterning."
+    # To Do: Get big cat sounds, maybe we can use the WoF leopard sounds?  
+    die_sound=drake-die.ogg
+    [abilities]
+        {ABILITY_AMBUSH}
+    [/abilities]
+    [movement_anim]
+        start_time=0
+        [frame]
+            image="units/monsters/cat/tritail-moving.png:150"
+        [/frame]
+    [/movement_anim]
+    {DEFENSE_ANIM "units/monsters/cat/tritail-defend-2.png" "units/monsters/cat/tritail-defend-1.png" {SOUND_LIST:DRAKE_HIT} }
+    [attack]
+        name=claws
+        description=_"claws"
+        range=melee
+        type=blade
+        damage=5
+        number=4
+        icon=attacks/claws-animal.png
+        attack_weight=0
+    [/attack]
+    [attack]
+        name=fangs
+        description=_"fangs"
+        icon=attacks/fangs-animal.png
+        type=blade
+        range=melee
+        damage=15
+        number=2
+        defense_weight=0
+    [/attack]
```

Comment:
It looks like you intend claws to be a defensive weapon and fangs to be an offensive weapon. It might be a matter of personal opinion, but I think it would make a little more sense to swap them.

I do wish attack/defense weight was implemented properly. I don't really agree with Ravana's notion that `[disable]` is better though – the engine should properly handle the weight function and convey it to the user if it's relevant.

Reply:
I was thinking the fangs are more of a pounce, the claws are a defensive batting away.  It's not like they ever really "select" a single weapon in any situation, but that's the wesnoth engine, and this would affect the animations.